vendor/modules.txt

```diff
@@ -13,12 +13,15 @@ github.com/PuerkitoBio/purell
 github.com/PuerkitoBio/urlesc
 # github.com/beorn7/perks v1.0.0
 github.com/beorn7/perks/quantile
-# github.com/datadog/extendeddaemonset v0.1.6 => ../ddaemonset/src/github.com/datadog/extendeddaemonset
+# github.com/datadog/extendeddaemonset v0.1.0
```

Comment:
This probably needs to be reverted... not sure why this change.

Reply:
Discussed with @clamoriniere this is actually correct, should be `v0.1.0`.

Reply:
Yes sorry I have committed some local change...